martin-mbtiles/src/tile_copier.rs

```diff
@@ -98,23 +98,40 @@ impl TileCopier {
             .execute(&mut conn)
             .await?;
 
-        let schema = query("SELECT sql FROM sourceDb.sqlite_schema WHERE tbl_name IN ('metadata', 'tiles', 'map', 'images')")
-            .fetch_all(&mut conn)
-            .await?;
-
-        for row in &schema {
-            let row: String = row.get(0);
-            query(row.as_str()).execute(&mut conn).await?;
-        }
+        if !self.options.force_simple {
```

Comment:
for slightly better ease-of-read, handle the "true" case first (here and below) -- there is even a lint for this. So just invert the if statement

Reply:
Right, conceptually I put the "default" case first, but I will switch them